modules/aws-limits.adoc

```diff
@@ -50,7 +50,7 @@ To use the `us-east-1` region, you must increase the EIP limit for your account.
 |Elastic Load Balancing (ELB/NLB)
 |3
 |20 per region
-|By default, each cluster creates internal and external network load balancers for the master API server and a single classic elastic load balancer for the router. Deploying more Kubernetes LoadBalancer Service objects will create additional link:https://aws.amazon.com/elasticloadbalancing/[load balancers].
+|By default, each cluster creates internal and external network load balancers for the API server and a single classic elastic load balancer for the router. Deploying more Kubernetes LoadBalancer Service objects will create additional link:https://aws.amazon.com/elasticloadbalancing/[load balancers].
```

Comment:
Do you need to differentiate between the primary (formerly master) and secondary API server here or is there always just one?